Source/Core/Core/PowerPC/JitILCommon/IR.cpp

```diff
@@ -1254,11 +1273,25 @@ InstLoc IRBuilder::FoldBiOp(unsigned Opcode, InstLoc Op1, InstLoc Op2, unsigned
 		case FDAdd:
 		case FDSub:
 			return FoldDoubleBiOp(Opcode, Op1, Op2);
+		case Store8:
+		case Store16:
+		case Store32:
+		case StoreSingle:
+		case StoreDouble:
+		{
+			auto mem = FoldMemoryAddress(Op2);
+			return EmitTriOp(Opcode, Op1, mem.first, mem.second);
```

Comment:
This is sort of abusing the intent of FoldBiOp etc.: given that Store8 is a three-operand instruction, it should be going through FoldTriOp.  (The difference doesn't really matter for now, but the intent was that we could merge blocks together by just refolding the IR, without going back to the original PPC asm.)